sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubConsumer.cs

```diff
@@ -62,6 +62,12 @@ public class EventHubConsumer : IAsyncDisposable
         ///
         public EventPosition StartingPosition { get; protected set; }
 
+        /// <summary>
+        ///   The text-based identifier label that has optionally been assigned to the consumer.
+        /// </summary>
+        ///
+        public string Identifier => Options?.Identifier;
```

Comment:
This was called out by @kinelski in the Live tests as an unintentional delta between track one and track two.